examples/play.yml

```diff
@@ -0,0 +1,522 @@
+#!/usr/bin/env ansible-playbook
+# ^ Trick: the above line can be used to make your play an executable
+# you also must add 'x' permissions to the file
+#
+# this file is based on phred's 'pedantically commented playbook'
+# https://github.com/phred/ansible-examples/blob/master/pedantically_commented_playbook.yml
+#
+---
+# ^^^ YAML documents can begin with the document separator "---"
+# and end with the "...", neither is needed for Ansible # as it does not
+# support multiple YAML documents per file, but some linters incorrectly insist
+# you must have it ....
+#
+# The '#' is a comment character, so any line starting with it will be ignored by Ansible.
+
+# Blank lines are ignored, so can be used # to create spacing to your taste.
+
+# Note about YAML: like Python, cares about whitespace, it requires actual spaces, tabs won't work.
+# Indent consistently throughout.
+# Two-space or four space indents is what most users prefer, but do whatever you like.
+#
+# If you're new to YAML, keep in mind that YAML documents, like XML
+# documents, represent a tree-like structure of nodes and text. More
+# familiar with JSON?  Think of YAML as a strict and more flexible JSON
+# with fewer significant characters (e.g., :, "", {}, []).
```

Comment:
This is confusingly worded. Both strict and more flexible? In what way does it have fewer significant characters?

Reply:
strict about spacing, more flexible about quoting:

``` {"this": "is valid json"} ```

```{ this: 'is valid for yaml, but not json'}```